contracts/ccip/GovernanceCCIPRelay.sol

```diff
@@ -3,60 +3,37 @@ pragma solidity ^0.8.20;
 
 import {IRouterClient} from "@chainlink/contracts-ccip/src/v0.8/ccip/interfaces/IRouterClient.sol";
 import {Client} from "@chainlink/contracts-ccip/src/v0.8/ccip/libraries/Client.sol";
+import {IGovernanceCCIPRelay} from "./interfaces/IGovernanceCCIPRelay.sol";
 
 /**
  * @title GovernanceRelay
  * @dev A contract for relaying governance proposals from Ethereum Mainnet to Destination Chain using CCIP.
  * This contract allows the Timelock contract to send cross-chain messages to a receiver contract on the destination chain..
  */
-contract GovernanceCCIPRelay {
-  /// @notice The CCIP router contract used for cross-chain messaging.
-  IRouterClient public ccipRouter;
+contract GovernanceCCIPRelay is IGovernanceCCIPRelay {
+  /// @inheritdoc IGovernanceCCIPRelay
+  IRouterClient public immutable ccipRouter;
 
-  /// @notice The address of the Timelock contract that can call this contract.
-  address public timelock;
+  /// @inheritdoc IGovernanceCCIPRelay
+  address public immutable timelock;
```

Comment:
@voith this is an immutable variable, replace the name in **uppercase** following the Solidity style recommendations